### ISSUE TITLE
An example of ancestor constructor usage in documentation

### ISSUE THREAD

**Issue by CowThing** [godotengine/godot#10211]
**bugsquad edit:** The issue was not real, but the documentation might be little misleading. Current docs example looks like this: 
```
func _init(args).(parent_args):
   pass
```
And basically that's what @CowThing was trying to do. Instead example in docs should look more like this I think:
```
func _init(args, parent_args).(parent_args):
   pass
```

-------

**Operating system or device - Godot version:**
Windows 10 - Godot 2.1.4 and Godot 3.0 alpha 1

**Issue description:**
If I have a base class that has arguments, and another class with arguments extends it, I get the following error:
`Parser Error: Identifier not found: a` on line 11 of the following script, which is `func _init(b).(a):`

In the [documentation on classes](http://docs.godotengine.org/en/stable/learning/scripting/gdscript/gdscript_basics.html#class-constructor) it mentions that parent arguments are passed like this:
```
func _init(args).(parent_args):
   pass
```

**Steps to reproduce:**
Add this script to a Node and run the project
```GDScript
extends Node

func _ready():
	var x = TestClass.new("A", "B")

class TestBase:
	func _init(a):
		print("a: ", a)

class TestClass extends TestBase:
	func _init(b).(a):
		print("b: ", b)
```

**Comment by eon-s** [comment]
TestClass `_init` have one parameter there, should be:

```gdscript 
func _init(arg_one, arg_two).(arg_one): #or arg_two
```

**Comment by CowThing** [comment]
Ah yes, you're correct, that fixed it. Maybe there should be an example use of that in the documentation, because the way it was described it didn't seem like that would have been the way to do it.

**Comment by kubecz3k** [comment]
Checked the docs, they indeed looks misleading, leaved the comment in the fist post.

**Comment by akien-mga** [comment]
This will be fixed for 4.0 via godotengine/godot-docs#3623.
